lib/shared/user_avatar.dart

```diff
@@ -24,6 +24,8 @@ class UserAvatar extends StatelessWidget {
           ),
         ));
 
+    if (person?.avatar == null || person!.avatar!.isEmpty) return placeholderIcon;
```

Comment:
Same comment about simplifying this check.

Reply:
Addressed in 819d861d04d66f2f1699d94c1a905cddeb931c38